package-lock.json

```diff
@@ -70,6 +70,12 @@
             "minimist": "^1.2.5"
           }
         },
+        "minimist": {
```

Comment:
Still changes. Could you try to simply replace this file in your branch with the original file and commit?

Reply:
oh, sorry, I copied contents from 2 previous commits instead of last commit, should be ok now.. 😅🦄

Reply:
Yes, looks good now, thanks